src/apps/backend/app.ts

```diff
@@ -42,6 +45,40 @@ export default class App {
     return Promise.resolve(server);
   }
 
+  public static getAPIMicroservices(): APIMicroserviceService[] {
+    const microservices: APIMicroserviceService[] = [];
+
+    // add your new server here to the list
+    const servers = [
```

Comment:
Can we move this in a separate function?

I think for all practical purposes, during application development, I expect this to change, and not rest of the code in this function.